test/callback-trigger-point.test.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @fileoverview Tests for the CallbackTriggerPoint.
+ */
+
+import {CallbackTriggerPoint} from '../src/lib/callback-trigger-point';
+
+describe('CallbackTriggerPoint', () => {
```

Comment:
Previously, there was no test for the `DrawTriggerPoint` that this class replaces.